cns/restserver/restserver.go

```diff
@@ -171,10 +171,24 @@ func (service *HTTPRestService) Start(config *common.ServiceConfig) error {
 	listener.AddHandler(cns.V2Prefix+cns.AttachContainerToNetwork, service.attachNetworkContainerToNetwork)
 	listener.AddHandler(cns.V2Prefix+cns.DetachContainerFromNetwork, service.detachNetworkContainerFromNetwork)
 
+	// Create ext network on windows platform if speficied through the cns commandline options.
+	// This allows orchestrators to start CNS which pre-provisions the network so that the
+	// VM network blip / disconnect is avoided when calling cni add for the very first time.
+	if err := service.CreateExtHnsNetwork(); err != nil {
+		log.Printf("[Azure CNS] Failed to create ext network. Error: %v", err)
+		return err
+	}
+
 	log.Printf("[Azure CNS]  Listening.")
 	return nil
 }
 
+// CreateExtHnsNetwork creates ext HNS network if not present.
+func (service *HTTPRestService) CreateExtHnsNetwork() error {
```

Comment:
lowercase (createExtHnsNetwork) if it isn't used elsewhere

Reply:
Done in latest update